packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/components/steps/link-user-archive/template.html

```diff
@@ -28,7 +28,7 @@
         <oui-button
             data-variant="secondary"
             data-icon-left="oui-icon-plus"
-            data-disabled="$ctrl.disabled"
+            data-disabled="$ctrl.disabled || $ctrl.isDiscoveryProject"
```

Comment:
How $ctrl.disabled is create, maybe you can implement isDiscoveryProject in same condition / function for avoid the condition test in html (if possible).

Reply:
A refactoring of this will be done in another PR.
I'll keep it in mind.